### PR TITLE
Handle FileAlreadyExistsException in ScalaPBGenerator

### PR DESCRIPTION
**Pull request by ptarjan**
workaround for https://github.com/bazelbuild/rules_scala/issues/779

### CODE REVIEW

**Comment by ptarjan** [comment]
@sholokhov can you comment `I consent.` in this PR please?

File: src/scala/scripts/ScalaPBGenerator.scala [hunk 1]
```diff
@@ -39,7 +40,13 @@ class ScalaPBGenerator extends Processor {
       val relativePath = root.relativize(fullPath)
 
       relativePath.toFile.getParentFile.mkdirs
-      Files.copy(fullPath, relativePath)
+      Try(Files.copy(fullPath, relativePath)) match {
+        case Failure(err) if err.isInstanceOf[FileAlreadyExistsException] =>
```
Comment (johnynek) [inline src/scala/scripts/ScalaPBGenerator.scala]: nit: `case Failure(err: FileAlreadyExistsException) => ` would be a bit less verbose.

File: src/scala/scripts/ScalaPBGenerator.scala [hunk 2]
```diff
@@ -39,7 +40,13 @@ class ScalaPBGenerator extends Processor {
...
+          Console.println(s"File already exists, skipping: ${err.getMessage}")
+        case Failure(err) =>
+          sys.error(err.getMessage)
```
Comment (johnynek) [inline src/scala/scripts/ScalaPBGenerator.scala]: nit, I would rethrow here: `throw err`

**Comment by sholokhov** [comment]
I consent.

**Comment by johnynek** [comment]
@iirina could you help us see how to satisfy the CLA bot. @ittaiz and I no longer have that power.

**Comment by ptarjan** [comment]
@johnynek given that he already said that, can I close this PR, and open a new one with me as the only author on the code?

**Pull request closed without merging**